mango/tests/processing_module/test_file_functions.py

```diff
@@ -255,7 +316,21 @@ def test_write_csv(self):
         write_csv(file, data)
         data2 = load_csv(file)
         self.assertEqual(data2.shape, (3, 2))
-        os.remove(file)
```

Comment:
Any reason why this line has to be removed?

Reply:
Because I use "file" in the row 326.

Reply:
It should be placed then in line 334? To avoid creating temporal useless files

Reply:
Done!